src/app/lib/access/forced_update/minimum_version_repository_mock.dart

```diff
@@ -0,0 +1,25 @@
+import 'dart:async';
+
+import 'package:app/access/forced_update/data/version.dart';
+import 'package:app/access/forced_update/minimum_version_repository.dart';
+import 'package:get_it/get_it.dart';
+
+/// Mock implementation of [MinimumVersionRepository].
```

Comment:
Was this mocked like that because it's not for tests ?

Reply:
Yes. And the reason it's mocked is the real implementation will need to use a backend.